deps/prebuilt/include/TAEF/WexTestClass.h

```diff
@@ -784,14 +864,54 @@ namespace WEX
 ///          TEST_CLASS(TestFeatureClass)
 ///      }
 /// \endcode
+#if defined (__clang__)
 #define TEST_CLASS(className) \
+protected: \
+    static constexpr auto c_taefClassName = L# className; \
```

Comment:
You shouldn't need both c_taefClassName and c_taefClassNameArray.

c_taefClassNameArray should be the one to use - it's a C++ array, but can be used to initialize a const wchar_t* (which is what it looks like you were using c_taefClassName for).